### PR TITLE
Update example to modern Kaleidoscope

### PR DESCRIPTION
**Pull request by ghoetker**
Makes changes to the example .ino file to match those made in the README as part of pull request #8 .

### CODE REVIEW

File: examples/DigitalRain/DigitalRain.ino [hunk 1]
```diff
@@ -4,7 +4,6 @@
 #include "Kaleidoscope-LEDControl.h"
 #include "Kaleidoscope-LEDEffect-DigitalRain.h"
 
-#include "LED-Off.h"
```
Comment (tremby) [inline examples/DigitalRain/DigitalRain.ino:7]: `LEDOff` is specified later in this file. Is this include really not needed?

If `LEDOff` isn't required, on the other hand, leave this removed (and remove one more newline please so there aren't three `\n`s in a row) and remove it below.

If it is required/useful, maybe add a comment saying why you'd want/need the LEDs to switch off?

File: examples/DigitalRain/DigitalRain.ino [hunk 2]
```diff
@@ -29,10 +28,24 @@ KEYMAPS(
...
+KALEIDOSCOPE_INIT_PLUGINS(
+
+  // LEDControl provides support for other LED modes
+  LEDControl,
+
+  // We start with the LED effect that turns off all the LEDs.
+  LEDOff,
+
+  // Support for DigitalRain
+   LEDDigitalRainEffect
```
Comment (tremby) [inline examples/DigitalRain/DigitalRain.ino]: Could you fix the indentation here please?

File: examples/DigitalRain/DigitalRain.ino [hunk 3]
```diff
@@ -29,10 +28,24 @@ KEYMAPS(
...
+
+  );
+
+  
 void setup() {
   Kaleidoscope.setup();
 
-  Kaleidoscope.use(&LEDControl, &LEDDigitalRainEffect, &LEDOff);
+  LEDDigitalRainEffect.DROP_TICKS = 22; // Make the rain fall faster
```
Comment (tremby) [inline examples/DigitalRain/DigitalRain.ino]: I think no need to change the configuration here in this file -- having it in the readme is enough, and this example file can be a bare minimum -- so remove this line (and the whitespace above) please.

**Review by tremby — CHANGES_REQUESTED** [review]
Thanks for this.

File: examples/DigitalRain/DigitalRain.ino (hunk 1 above)
Comment (ghoetker) [reply examples/DigitalRain/DigitalRain.ino:7]: LED-off functionality is now part of the core Kaleidoscope include.  However, to make it active, thus allowing a setting with LED's office, LEDOff must be included in the Plugins. In the name of making a minimal example, I've removed the LEDOff.

File: examples/DigitalRain/DigitalRain.ino (hunk 2 above)
Comment (ghoetker) [reply examples/DigitalRain/DigitalRain.ino]: Done.

File: examples/DigitalRain/DigitalRain.ino (hunk 3 above)
Comment (ghoetker) [reply examples/DigitalRain/DigitalRain.ino]: Done.

**Comment by ghoetker** [comment]
Requested changes made as noted above.

**Comment by tremby** [comment]
Thank you!

**Pull request merged**